tests/Base.php

```diff
@@ -57,6 +57,11 @@ protected function _getClient(array $params = [], ?callable $callback = null, ?L
 
         $config = \array_merge($config, $params);
 
+        // TODO: either find a way to automatically fetch username/pw for ES, or find a way to disable the security
+        // In ES8, security is enabled by default
+        $config['username'] = 'elastic';
+        $config['password'] = \getenv('ES_PASSWORD');
```

Comment:
For testing purpose, I suggest we just set the password in the elasticsearch container to `changeme` and also hardcode it here.

Reply:
I hadn't noticed there was a docker container :) I'll add it to the tasklist!